platform/jobs/edxPlatformQualityMaster.groovy

```diff
@@ -33,7 +33,7 @@ archiveReports += 'edx-platform*/test_root/log/*.log, edx-platform*/test_root/lo
 archiveReports += 'edx-platform*/**/nosetests.xml,edx-platform*/**/TEST-*.xml'
 
 String htmlReports = 'pylint/*view*/, pep8/*view*/, jshint/*view*/, python_complexity/*view*/,'
-htmlReports += 'safecommit/*view*/, safelint/*view*/, eslint/*view*/'
+htmlReports += 'xsscommitlint/*view*/, xsslint/*view*/, eslint/*view*/'
```

Comment:
@estute I think it is important to not break builds, but it is not important to have these tabs during the transition.  Would you be ok with me removing the tabs in one PR and then in a week or two adding them with the new names in a new PR?  I want to keep the cost down on this change.